### DISCUSSION TITLE
Failed to fetch https://packages.microsoft.com/repos/edge/dists/stable/main/binary-amd64/Packages.gz  File has unexpected size (8078 != 8155). Mirror sync in progress? [IP: 13.107.213.51 443]

### DISCUSSION THREAD

**Discussion by VietND96** [community/community#121059] [Actions] [answered]
### Select Topic Area

Question

### Body

I am not sure this is the right channel to ask this issue.
The issue is Action randomly failed with the below reason

>
2.153 E: Failed to fetch https://packages.microsoft.com/repos/edge/dists/stable/main/binary-amd64/Packages.gz  File has unexpected size (8078 != 8155). Mirror sync in progress? [IP: 13.107.213.51 443]
2.153    Hashes of expected file:
2.153     - Filesize:8155 [weak]
2.153     - SHA512:c2a3e7edbc44eac190f8df014238d836992b1300d52ac5eed89dc9cc571c1e35623a281e9544649418ec167a854f707a407f3c252076282fe3ba61049f835cd3
2.153     - SHA256:41e205cfa4dc4dce04cc0ed0cae41245955ba6f49fcd331d55a712e97c4a8bf8
2.153    Release file created at: Thu, 18 Apr 2024 18:02:47 +0000
2.153 E: Some index files failed to download. They have been ignored, or old ones used instead.

I think the Mirror would be selected randomly based on the host region or something else. Can we consider checking this mirror address? Or exclude the fault mirror from the list?

**Comment by Titas-OpenTrack** [comment, marked as answer]
As per [documentation](https://learn.microsoft.com/en-us/dotnet/core/install/linux-debian#failed-to-fetch), since it's been happening for more than 30 minutes, I have opened an [issue](https://github.com/dotnet/core/issues/9284) in the appropriate repo.
